src/Symfony/Component/Security/Core/Security.php

```diff
@@ -22,4 +26,50 @@
     const AUTHENTICATION_ERROR = '_security.last_error';
     const LAST_USERNAME = '_security.last_username';
     const MAX_USERNAME_LENGTH = 4096;
+
+    private $container;
+
+    public function __construct(ContainerInterface $container)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * @return UserInterface|null
+     */
+    public function getUser()
+    {
+        if (!$token = $this->getToken()) {
+            return;
+        }
+
+        $user = $token->getUser();
+        if (!is_object($user)) {
+            return;
```

Comment:
as of PHP 7.1 we have the void return type ([reference](http://php.net/manual/en/migration71.new-features.php#migration71.new-features.void-functions)) so I suggest making the null explicit:

```php
return null;
```